src/stateManagement/textStyle.js

```diff
@@ -1,38 +1,110 @@
-let defaultFontSize = 15,
-  defaultFont = `${defaultFontSize}px Arial`,
-  defaultBackgroundColor = 'transparent';
+const textStyle = {
+  fontSize: 15,
+  fontFamily: 'Arial',
+  backgroundColor: 'transparent',
+};
 
+/**
+ * @deprecated
+ * This function will be deprecated and users must use `setFontSize` and
+ * `setFontFamily` to set the font configuration independently.
+ *
+ * @param {string} font Font string with font size and font family
+ * @returns {void}
+ */
 function setFont(font) {
-  defaultFont = font;
+  const split = font.split('px ');
+
+  if (split.length === 2) {
+    setFontSize(split[0]);
+    setFontFamily(split[1]);
+  }
 }
 
+/**
+ * Returns all the font styling properties combined into a single string
+ * following the CSS font property pattern.
+ * @see {@link https://www.w3.org/TR/css-fonts-3/#font-prop|3.7. Shorthand font property: the font property}
+ *
+ * @returns {string} CSS font property
+ */
 function getFont() {
-  return defaultFont;
+  return `${textStyle.fontSize}px ${textStyle.fontFamily}`;
+}
+
+/**
+ * Sets the font family that will be used for general font rendering.
+ * @see {@link https://www.w3.org/TR/css-fonts-3/#font-family-prop|3.1. Font family: the font-family property}
+ *
+ * @param {string} fontFamily Name of the font to be used
+ * @returns {void}
+ */
+function setFontFamily(fontFamily) {
+  textStyle.fontFamily = fontFamily;
```

Comment:
what happens if i `setFontFamily` to null or undefined or non-standard font names?

Reply:
I think this is the user's responsibility? maybe adding checks could lead to silently broken code. @brunoalvesdefaria any thoughts on this?